crescent_bot/__main__.py

```diff
@@ -0,0 +1,32 @@
+import os
+from pathlib import Path
+
+import hikari
+import crescent
+
+
+def create_bot() -> crescent.Bot:
+    # Load the token from a secrets file you'll need to create yourself.
+    with open("./secrets/token") as f:
+        token = f.read().strip()
+
+    # Create the main bot instance with all intents.
+    bot = crescent.Bot(token, intents=hikari.Intents.ALL_UNPRIVILEGED)
```

Comment:
Iirc the other examples use ALL, including privileged. If I'm wrong just ignore this.